prime-router/src/test/kotlin/ReceiverTests.kt

```diff
@@ -273,4 +274,11 @@ internal class ReceiverTests {
         val receiver = Receiver("elr", "IGNORE", Topic.COVID_19, CustomerStatus.INACTIVE, translatorConfig)
         assertThat(receiver.transportType.type).isEqualTo("NULL")
     }
+
+    @Test
+    fun `test MAJURO receiver timezone`() {
+        val shouldWork1 =
+            ZonedDateTime.of(2024, 12, 20, 0, 0, 0, 0, ZoneId.of(USTimeZone.MAJURO.zoneId)).toOffsetDateTime()
+        assertThat(shouldWork1).isNotEqualTo("2024-12-20T00:00+12:00")
+    }
```

Comment:
What is this test actually testing? Is testing that it works or that it doesn't work?

Reply:
Test to see, it is working.

Reply:
Shouldn't it be testing that the date gets converted to MAJURO than?

Reply:
Oh yes, are you correct.  I will test the date get converted to MAJURO as well.  Thanks.

Reply:
Added the test the date get converted to MAJURO.  Done.